### DISCUSSION TITLE
Path to standardisation!

### DISCUSSION THREAD

**Discussion by Relequestual** [json-schema-org/community#3] [Vision and direction]
How? When? Where?
These are questions that have been asked about JSON Schema and standardisation many times.
We now want to assess possible directions.

There have been some really great discussions, and I want to link to as many as possible from here.

I'll also be creating a new discussion to track specific possible paths.

It's worth noting that joining a foundation as a project does not prohibit us publishing the specification to any specific standards organisation if we choose to do so. In this I mean, if we choose to join the OpenJS Foundation, we aren't prohibited from still publishing specification documents through the IETF.

I'll try to keep this top post updated with the current status, as well as posting the same content as follow up comments.

**Comment by Relequestual** [comment]
A discussion relating to the option to self publish: https://github.com/json-schema-org/community/discussions/69

**Comment by gregsdennis** [comment]
Discussion of publishing a specification with the purpose of moving it out of "draft" status an into an initial state of stability: #282

**Comment by brontobrit** [comment]
Hi!

There are several SMPTE (www.smpte.org) Standards projects that need a standardized JSON Schema Language. Most notable is SMPTE ST2135 RegJSON whose goal is a canonical isomorphic mapping between JSON and SMPTE ST336 KLV (binary) Encoding. KLV is the underpinnings of SMPTE ST377 MXF which is the predominant format for professional media files.

I am the Editor of ST2135 RegJSON. Previously I was the author of KLV and the creator of MXF and chaired the committees that brought all these Standards to publication. My long time colleague Bruce Devlin is the chair of the RegJSON project.

In an ideal world, ST2135 RegJSON would be able to normatively reference an RFC or a W3C specification based on your excellent work. If that is not your path, we have also contemplated if it would be possible to publish your work in our proposed Standard.

Please forgive my newness to json-schema.org and help me to address my questions to the right people in the right forums (here? slack? direct email? voice?)

best regards
Oliver Morgan
oliver@metaglue.com

**Reply by Relequestual** [reply]
Hey Oliver. Thanks for reaching out. This form is just fine.

There are several other in progress standards which have similar questions. Here's one I found just now: https://github.com/w3c/vc-json-schema/issues/133

You may find it interesting that we are currently in the process of defining "compatible release era" or "stable" JSON Schema (See [decision record](https://github.com/json-schema-org/json-schema-spec/blob/main/adr/2022-11-stable-spec.md)). The discussion is still ongoing, but we are aiming for the next version of JSON Schema to be a foundation for all future versions in terms of being compatible. There's still a [fair amount of work](https://github.com/json-schema-org/community/issues/415) to be done. We do not have a timeframe.

Will we eventually try to publish something minimal to a standards org? It's possible, but not a priority. We only recently decided to [move away from the IETF draft publication process](https://json-schema.org/blog/posts/future-of-json-schema).

The JSON Schema project is a part of the OpenJS Foundation, a Linux Foundation project. The LF also has an Open Standards group, which we may seek to join at some point.

Today, JSON Schema is an industry standard, and it is baked in to the most popular IDEs and code editors. We also have [13 published case studies](https://github.com/orgs/json-schema-org/projects/8/views/2). I only share this as I think it speaks to the potential for an exemption for normative referencing, especially given multiple draft standards are wanting to (per the first link).

Feel free to reply here and/or join our slack server.

**Comment by brontobrit** [comment]
Ben:

Thanks for your prompt reply.
Standards bodies generally only normatively reference specs published by an accredited SDO, exceptions are not possible. per ISO rules.

That said, it might be possible to count LF as such. I can ask SMPTE ST.

>Will we eventually try to publish something minimal to a standards org? It's possible, but not a priority.
Understood. Looking at this differently, if adequate resources became available, and the new resources thought it was high priority, could the steering group reconsider?

I completely understand that the SDO process is consuming and distracts from work towards tangible projects. So it needs a dedicated resource. 

> something minimal
This also sounds attractive. I'd like to discuss what the minimal outline would need to cover .

thanks
Oliver
